README.md

```diff
@@ -84,6 +84,8 @@ myGraph.tickFrame();
 | <b>emitParticle</b>(<i>link</i>) | An alternative mechanism for generating particles, this method emits a non-cyclical single particle within a specific link. The emitted particle shares the styling (speed, width, color) of the regular particle props. A valid `link` object that is included in `graphData` should be passed as a single parameter. ||
 | <b>forceEngine</b>([<i>str</i>]) | Getter/setter for which force-simulation engine to use ([*d3*](https://github.com/vasturiano/d3-force-3d) or [*ngraph*](https://github.com/anvaka/ngraph.forcelayout)). | `d3` |
 | <b>d3ReheatSimulation</b>() | Reheats the force simulation engine, by setting the `alpha` value to `1`. Only applicable if using the d3 simulation engine. | |
+| <b>d3Alpha</b>([<i>num</i>]) | Getter for the [simulation alpha](https://github.com/vasturiano/d3-force-3d#simulation_alpha) parameter, only applicable if using the d3 simulation engine. | `1` |
```

Comment:
I don't think we need to expose `d3Alpha` externally. That should remain an internal parameter of the engine. One should be able to calibrate the engine behavior strictly from the alpha decay and min properties.

Reply:
They idea behind it was to have a getter so the user can have more information about the state of the graph.

Reply:
From the doc line it looks like a setter (you can specify a `number`) which is disadvisable. I'm still leaning towards not exposing this prop since it's really a system internal.

Reply:
Actually the only reason i added it was so in [the 3d-force-graph PR](https://github.com/vasturiano/3d-force-graph/pull/309) the user could see that value, like kind of a very poor test.

I don't mind removing it, but also it doesn't harm the user having that extra information.

Reply:
Not to snowball the API, prefer to leave it out until there's a compelling use case.